src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -157,123 +161,130 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
+Che CLI allows a wide range of configuration changes to setup the port, hostname, oAuth, Docker, git, and to solve networking issues. For details, see link:docker-config.html[Che configuration on Docker].
 
 [id="run-without-cli"]
-== Run Without CLI
+== Running Che without the CLI
+
+You can run the Che server directly by launching a Docker image. This approach bypasses the CLI that has additional utilities to simplify administration and operation. The `eclipse/che-server` Docker image is appropriate for running Che within clusters, orchestrators, or by third-party tools with automation.
 
-You can run the Che server directly by launching a Docker image. This approach bypasses the CLI, which has additional utilities to simplify administration and operation. The `eclipse/che-server` Docker image is appropriate for running Che within clusters, orchestrators, or by third party tools with automation.
+To run Che directly by launching a Docker image, take the following steps:
 
+. Run the latest released version of Che.
+. Replace <LOCAL_PATH> with any host folder. Che places the backup files in this folder - configurable properties, workspaces, lib, storage.
++
 ----
-# Run the latest released version of Che
-# Replace <LOCAL_PATH> with any host folder
-# Che will place backup files there - configurable properties, workspaces, lib, storage
 docker run -p 8080:8080 \
            --name che \
            --rm \
            -v /var/run/docker.sock:/var/run/docker.sock \
            -v <LOCAL_PATH>:/data \
            eclipse/che-server:6.0.0
+----
 
-# To run the nightly version of Che, replace eclipse/che-server:5.0.0-latest with
-eclipse/che-server:nightly
+Following are some useful commands:
 
-# To run a specific tagged version of Che, replace eclipse/che-server:5.0.0-latest with
-eclipse/che-server:<version>
+* To run the nightly version of Che, replace `eclipse/che-server:5.0.0-latest` with `eclipse/che-server:nightly`.
 
-# Stop the container running Che
-docker stop che
+* To run a specific tagged version of Che, replace `eclipse/che-server:5.0.0-latest` with `eclipse/che-server:<_version_>`.
 
-# Restart the container running Che and restart the Che server
-docker restart che
+* To Stop the container running Che, use the `docker stop che` command.
+
+* To restart the container running Che and restart the Che server, use the `docker restart che` command.
 
-# Upgrade to a newer version
+* To upgrade to a newer version, use the following commands in order:
++
+----
 docker pull eclipse/che-server:6.0.0-latest
 docker restart che
 ----
 
-Che has started when you see `Server startup in ##### ms`. After starting, Che is available at `localhost:8080` or a remote IP if Che has been started remotely.
+The *Server startup in ##### ms* message confirms that Che has started. After starting, Che is available at `localhost:8080` or at a remote IP if you started Che remotely.
 
 *SELinux*
```

Comment:
* -> section heading
* -> action title

Reply:
Changed to "Running Che on SELinux"

Reply:
Better "with SELinux" -- it's a tool, not a platform.